File: docs/legacy(old PACE)/unsupported/VB.NET/PaceWebServiceSDK/sample/FindOpenJobs.vb
Imports PaceWebServiceSDK.efipaceservices

Public Class FindOpenJobs

    Public Shared Sub Run()
        Dim findObjectsHttpBinding As FindObjectsHttpBinding = PaceClient.getFindObjectsHttpBinding()
        Dim keys As String() = findObjectsHttpBinding.find("Job", "adminStatus/@openJob")
        Console.WriteLine(keys.Length.ToString + " Open Jobs")
        Console.WriteLine(String.Join(",", keys))
    End Sub

End Class

File: docs/legacy(old PACE)/C#/4.0/PaceAPISample/sample/FindOpenJobs.cs
using System;

using efipaceservices;

namespace Pace_Web_Service_SDK
{
	/// <summary>
	/// Summary description for FindOpenJobs.
	/// </summary>
	public class FindOpenJobs
	{
		public static void Run()
		{
            FindObjectsHttpBinding findObjectsHttpBinding = PaceClient.getFindObjectsHttpBinding();

            string[] keys = findObjectsHttpBinding.find("Job", "adminStatus/@openJob");
			Console.WriteLine( keys.Length + " Open Jobs" );
			Console.WriteLine( string.Join(",", keys) );
		}
	}
}
